### DISCUSSION TITLE
vel_gen

### DISCUSSION THREAD

**Discussion by slamdunk1002** [rodralez/NaveGo#51] [Q&A] [answered]
Hello, Dr.
   your vel_gen.m file  lack of function diff__central().
   I don't understand how to get.
   Can you tell me? Thank you.

**Comment by rodralez** [comment, marked as answer]
Hello,

Yes, this file was missing. My mistake. Thank you for pointing out this problem.

The file was restored at the simulation directory.

Kind regards.
